### PR TITLE
Add README.txt

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: README.txt [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+## Dependencies
+
+A liver
+
+## How to contribute
+
+1. Read our Guiness Challenge Style Guide on Notion
+2. Go to The Engineer after Harker Tech every week
+3. Do the Guiness Challenge in line with our style guide
+4. Upload a picture of your challenge result with the `.JPEG` file extension (`.JPEG` specifically, none of this cop-out `.JPG` behaviour)
```
Comment (ricochetmoo) [inline README.txt]: @faizanabbasjsc we decided at the All Hands on 25/06/2024 that the standard for Guinness attempts will be GIFs. Please amend accordingly. It's worth keeping in the details about JPEGs anyway, just to idiot-proof it.

**Review by ricochetmoo — CHANGES_REQUESTED** [review]
👎 not a great start.

File: README.txt (hunk 1 above)
Comment (ghost) [reply README.txt]: Apologies this slipped my mind, I shall make this change with haste.

**Review by ricochetmoo — APPROVED** [review]
Much better. All approved. 👍
Happy drinking!

**Pull request merged**